components/Slider/Slider.js

```diff
@@ -42,6 +36,12 @@ export default class Slider {
           </div>
           : null
         }
+        <ReactSlider withBars
```

Comment:
We can roll with this one as it seems to work. Change it up later if theres an issue.